gradoop-flink/src/main/java/org/gradoop/flink/algorithms/gelly/trianglecounting/functions/WritePropertyToGraphHeadMap.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.algorithms.gelly.trianglecounting.functions;
+
+import org.apache.flink.api.common.functions.MapFunction;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+import org.gradoop.common.model.impl.properties.PropertyValue;
+
+/**
+ * Writes the given property value to the graph head.
+ */
+public class WritePropertyToGraphHeadMap implements MapFunction<GraphHead, GraphHead> {
```

Comment:
Move this class to `org.gradoop.flink.algorithms.gelly.functions` and use it also in your other PR. I'll put a comment there too.